site/_config.yml

```diff
@@ -32,3 +32,7 @@ gems:
   - jekyll-seo-tag
   - jekyll-avatar
   - jekyll-mentions
+
+exclude:
+  - README.md
+  - .gitignore
```

Comment:
@crispgm, currently the files [here](https://github.com/jekyll/jekyll/tree/master/lib/site_template) plua a Gemfile are copied to a new site's root by `jekyll new`